sdk/appconfiguration/azure-appconfiguration/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 ### Bugs Fixed
 
 ### Other Changes
+- Bumped minimum dependency on `azure-core` to `>=1.28.0`.
```

Comment:
nit: you can just say "...to `1.28.0`

Reply:
I'll keep it to align with the similar updates in history.